packages/preview/unify/0.6.0/lib.typ

```diff
@@ -0,0 +1,744 @@
+#let re-num = regex("^(-?\d+(\.|,)?\d*)?(((\+(\d+(\.|,)?\d*)-(\d+(\.|,)?\d*)))|((((\+-)|(-\+))(\d+(\.|,)?\d*))))?(e([-\+]?\d+))?$")
+#let unicode_exponents = (("\u2070", "0"), ("\u00B9", "1"), ("\u00B2", "2"), ("\u00B3", "3"), ("\u2074", "4"), ("\u2075", "5"), ("\u2076", "6"), ("\u2077", "7"), ("\u2078", "8"), ("\u2079", "9"), ("\u207A", "+"), ("\u207B", "-"))
```

Comment:
I'm not sure if this variable is intended for public use or not, but if not it should probably be indicated by adding a `_` in front as you did with other variables and functions. And if it is, it should probably use kebab-case as the rest of the package.